src/AWS.Deploy.Orchestration/CdkProjectHandler.cs

```diff
@@ -27,6 +27,8 @@ public CdkProjectHandler(IOrchestratorInteractiveService interactiveService, ICo
 
         public async Task CreateCdkDeployment(OrchestratorSession session, CloudApplication cloudApplication, Recommendation recommendation)
         {
+            var recipeInfo = $"{recommendation.Recipe.Id}_{recommendation.Recipe.Version}";
```

Comment:
we should think about storing this info globally without relying on the developer to pass this every time they do a Run in CommandLineWrapper

Reply:
recipe info is passed only for `cdk deploy`, we don't have any other-use also where we want to pass it.

See https://github.com/aws/aws-dotnet-deploy/pull/184/files#r610240199 for shared state.